packages/datatrak-web/src/layout/MainPageLayout.tsx

```diff
@@ -6,18 +6,22 @@
 import React from 'react';
 import { Outlet } from 'react-router';
 import styled from 'styled-components';
+import { HEADER_HEIGHT } from '../constants';
 import { Header } from '.';
 
 const PageWrapper = styled.div`
   display: flex;
   flex-direction: column;
   background: ${({ theme }) => theme.palette.background.default};
   min-height: 100vh;
+  .notistack-SnackbarContainer {
+    top: calc(1rem + ${HEADER_HEIGHT});
+  }
 `;
 
 export const MainPageLayout = () => {
   return (
-    <PageWrapper>
+    <PageWrapper id="page">
```

Comment:
What is the id for?

Reply:
It's so that we can tell the toast messages to sit within this container, instead of being on top of the header

Reply:
ah yes I just saw that in the other file. Makes sense. Would you mind adding a comment in this file too. thanks